.env

```diff
@@ -108,6 +108,13 @@ HASH_TELEMETRY_DESTINATION=REPLACE_ME.aws.com
 # Is used for differentiating different apps, can be any value
 HASH_TELEMETRY_APP_ID=hash-app
 
+# Disable telemetry from third-party dependencies who transmit IP addresses
+NEXT_TELEMETRY_DISABLED=1 # Vercel Next.js
+TURBO_TELEMETRY_DISABLED=1 # Vercel Turborepo
+YARN_ENABLE_TELEMETRY=0 # Yarn
+ARTILLERY_DISABLE_TELEMETRY=true # Artillery
+CHECKPOINT_DISABLE=1 # Terraform and others
+
```

Comment:
In most cases setting these here has no effect as `.env` is not read as part of it. This may leads to confusing that we specify these variables, however, they appear to not work.

In our applications, we specifically invoke `dotenv-flow`, which loads this and other `.env` files, however, if we just run a command like `turbo` this won't be seen.

Also important to notice that this _will not_ have any impact on the production instance.

Reply:
Following up with you separately.